### PR TITLE
[Component] Set overlays via the API

### PR DESCRIPTION
**Pull request by liorze**

### CODE REVIEW

File: src/app/@ansyn/ansyn/app-effects/effects/core.app.effects.ts [hunk 1]
```diff
@@ -80,6 +80,7 @@ export class CoreAppEffects {
 	@Effect()
 	loadOverlays$ = this.actions$
 		.ofType<SetOverlaysCriteriaAction>(CoreActionTypes.SET_OVERLAYS_CRITERIA)
+		.filter(action => action.payload.search !== false)
```
Comment (asafMasa) [inline src/app/@ansyn/ansyn/app-effects/effects/core.app.effects.ts]: add comment // This is a hack, if undefined don't cancel search

File: src/app/builder/api/ansyn-api.service.ts [hunk 2]
```diff
@@ -80,6 +80,10 @@ export class AnsynApi {
 		this.store.dispatch(new DisplayOverlayAction({ overlay, mapId: this.activeMapId, forceFirstDisplay: true }));
 	}
 
+	setOverlays(overlays: IOverlay[]) {
```
Comment (asafMasa) [inline src/app/builder/api/ansyn-api.service.ts]: Update https://github.com/AnSyn/ansyn-angular1/wiki/Ansyn-public-API#setoverlaysoverlays-overlay wiki
add NOTE : that if user wants to use this method he should call "disableInitialSearch" at constructor

File: src/app/builder/api/ansyn-api.service.ts [hunk 3]
```diff
@@ -88,6 +92,9 @@ export class AnsynApi {
 		this.store.dispatch(new SelectCaseAction(this.casesConfig.defaultCase));
 	}
 
+	disableInitialSearch() {
```
Comment (asafMasa) [inline src/app/builder/api/ansyn-api.service.ts]: add to above wiki

Comment (asafMasa) [inline src/app/builder/api/ansyn-api.service.ts]: rename to enableInitialSearch(isEnabled: string)
make it a private function
call with constructor arg

File: src/app/@ansyn/ansyn/app-effects/effects/core.app.effects.ts (hunk 1 above)
Comment (ramtob) [reply src/app/@ansyn/ansyn/app-effects/effects/core.app.effects.ts]: done

File: src/app/builder/api/ansyn-api.service.ts (hunk 2 above)
Comment (ramtob) [reply src/app/builder/api/ansyn-api.service.ts]: Added "I" to interfaces names

**Pull request merged**